cli/ballerina-cli/src/main/resources/create_cmd_templates/service/service.bal

```diff
@@ -1,21 +1,19 @@
 import ballerina/http;
-import ballerina/io;
 
 # A service representing a network-accessible API
-# bound to absolute path `/hello` and port `9090`.
-service /hello on new http:Listener(9090) {
+# bound to port `9090`.
+service hello on new http:Listener(9090) {
 
     # A resource respresenting an invokable API method
     # accessible at `/hello/sayHello`.
-    #
-    # + caller - the client invoking this resource
-    # + request - the inbound request
-    resource function get sayHello(http:Caller caller, http:Request request) {
-
+    # add return
+    resource function sayHello(string name) returns string {
         // Send a response back to the caller.
-        error? result = caller->respond("Hello Ballerina!");
-        if (result is error) {
-            io:println("Error in responding: ", result);
+
+        if (name.isBlank()) {
```

Comment:
A method called `isBlank()` does not exist AFAIK.

Reply:
This was suggested by @Dilhasha in https://github.com/ballerina-platform/ballerina-lang/pull/31335#discussion_r663669492